logstash-core/logstash-core.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = LOGSTASH_CORE_VERSION
 
-  gem.add_runtime_dependency "logstash-core-event-java", "~> 3.0.0.dev"
+  gem.add_runtime_dependency "logstash-core-event-java", "2.3.0.snapshot1"
```

Comment:
han. I don't know anymore :/  - let's clarify our core gems versions constrains strategy. I think that for the snapshotX it doesn't matter much and we can keep a fixed version constrain since we will always release new complete snapshots